From dfb23b60727a7cf1703b1ab4ce0f474faf65b9b1 Mon Sep 17 00:00:00 2001
From: Zhou weixin <zwx@rock-chips.com>
Date: Thu, 3 May 2018 11:49:06 +0800
Subject: [PATCH] arm64: dts: rockchip: disable kpti for px30

The cpu will block if the CONFIG_RANDOMIZE_BASE\rockchip-dmc\kpti
are enabled, and the kpti will reduced system performance.

Change-Id: Icd054a4a9cbbea20bd8f8c33a6db64c6fd304fd0
Signed-off-by: Weixin Zhou <zwx@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/px30-android.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/px30-android.dtsi b/arch/arm64/boot/dts/rockchip/px30-android.dtsi
index 1dc6ff5d9ad9..c9674e77e2b3 100644
--- a/arch/arm64/boot/dts/rockchip/px30-android.dtsi
+++ b/arch/arm64/boot/dts/rockchip/px30-android.dtsi
@@ -6,7 +6,7 @@
 
 / {
 	chosen {
-		bootargs = "earlyprintk=uart8250,mmio32,0xff160000 swiotlb=1 console=ttyFIQ0 androidboot.baseband=N/A androidboot.selinux=permissive androidboot.veritymode=/dev/block/by-name/metadata androidboot.hardware=rk30board androidboot.console=ttyFIQ0 init=/init";
+		bootargs = "earlyprintk=uart8250,mmio32,0xff160000 swiotlb=1 console=ttyFIQ0 androidboot.baseband=N/A androidboot.selinux=permissive androidboot.veritymode=/dev/block/by-name/metadata androidboot.hardware=rk30board androidboot.console=ttyFIQ0 init=/init kpti=0";
 	};
 
 	fiq-debugger {
-- 
2.35.3

